src/toil/test/utils/ABC/debugWorkflow.py

```diff
@@ -0,0 +1,177 @@
+from toil.job import Job
+from toil.common import Toil
+import subprocess
+import os
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def initialize_jobs(job):
+    job.fileStore.logToMaster('''initialize_jobs''')
+
+def writeA(job, mkFile):
+    job.fileStore.logToMaster('''writeA''')
+
+    # temp folder for the run
+    tempDir = job.fileStore.getLocalTempDir()
+
+    # import files
+    try:
+        mkFile_fs = job.fileStore.readGlobalFile(mkFile[0], userPath=os.path.join(tempDir, mkFile[1]))
+    except:
+        mkFile_fs = os.path.join(tempDir, mkFile[1])
+
+    # make a file (A.txt) and write 'A' into it using 'mkFile.py'
```

Comment:
It more closely approximates what toil is intended to do: call things outside of itself, and coordinate files outside of itself.  That's the intent, though maybe I should use one of the WDL files instead.  I thought this would be quicker.